<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryStreamer</class>
 <widget class="QWidget" name="BinaryStreamer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BinaryStreamer</string>
  </property>
  <widget class="QPlainTextEdit" name="ProgramOutputBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>781</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="RunButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>71</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>774</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Scan Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="ScanStartBox">
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Scan End</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="ScanEndBox">
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string># of Cycles</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="NumCyclesBox">
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Gas Steps</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="GasStepsBox">
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Frames</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="FrameBox">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="TypeBox">
      <property name="currentText">
       <string>Static</string>
      </property>
      <item>
       <property name="text">
        <string>Static</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dynamic</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Dark</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="DarkBox">
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Pump On</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="PumpOnBox">
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>781</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="InputButton">
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="InputField"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="OutputButton">
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="OutputField"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="LogField"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="LogButton">
      <property name="text">
       <string>Log</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="ClearButton">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SaveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string># of Steps</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="NumStepsBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Prefix</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PrefixBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>313</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCheckBox" name="RawTextCheckBox">
      <property name="text">
       <string>Raw Text Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="dODBox">
      <property name="text">
       <string>Calculate dOD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="AttoCubeCheck">
      <property name="text">
       <string>AttoCube</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Time Offset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="OffsetBox">
      <property name="minimum">
       <double>-500.000000000000000</double>
      </property>
      <property name="maximum">
       <double>500.000000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
